functions/api.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express'
+import { ApolloServer } from 'apollo-server-cloud-functions'
+import { buildSchema } from 'type-graphql'
+import { HelloWorldResolver } from './GraphQL/resolvers'
+
+export const APIServer = async (req: Request, res: Response) => {
```

Comment:
Functions should always start with a lower case letter. Eg. `apiServer`. `UpperCase` should be reserved for un-instantiated classes.